nessie/lib/queries.py

```diff
@@ -112,10 +112,22 @@ def undergrads_schema():
 
 
 def get_all_student_ids():
-    sql = f"""SELECT sid FROM {asc_schema()}.students
-        UNION SELECT sid FROM {coe_schema()}.students
-        UNION SELECT sid FROM {undergrads_schema()}.students
-        UNION SELECT sid FROM {advisee_schema()}.non_current_students"""
+    if feature_flag_edl():
+        sql = f"""SELECT student_id AS sid
```

Comment:
@johncrossman Do we need a `SELECT DISTINCT` here? I think the EDL table may sometimes have multiple rows for the same SID even with the restrictions below.

Reply:
@pauline2k - I assumed UNION would de-dupe the list but I'll add DISTINCT to be safe.